### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by z0rats**

### CODE REVIEW

File: .github/workflows/coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          node-version: ${{ matrix.node-version }}
+      
+      - name: Install dependencies
+        run: npm i
+
+      - name: Generate TypeChain artifacts
+        run: npx hardhat typechain --config hardhat-ci.config.ts
+        
+      - name: Run coverage
+        run: npx hardhat coverage --config hardhat-ci.config.ts
```
Comment (ioffikus) [inline .github/workflows/coverage.yaml]: new line @v0xat

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  overrides: [
+    {
+      files: ["*.test.ts", "*.spec.ts"],
+      rules: {
+        "no-unused-expressions": "off"
+      }
+    }
+  ]
+};
```
Comment (ioffikus) [inline .eslintrc.js]: new line @v0xat

**Review by ioffikus — CHANGES_REQUESTED** [review]

File: .github/workflows/coverage.yaml (hunk 1 above)
Comment (z0rats) [reply .github/workflows/coverage.yaml]: fixed

File: .eslintrc.js (hunk 2 above)
Comment (z0rats) [reply .eslintrc.js]: fixed

**Pull request merged**